flake8_import_order/stdlib_list.py

```diff
@@ -1,4 +1,4 @@
-STDLIB_NAMES = set((
+STDLIB_NAMES = {
```

Comment:
Has flake8-import-order dropped Python 2.6 support yet?

Reply:
I was under the impression they had.

Reply:
I confirmed the travis tests do not test for 2.6 and recently (4 months ago) 3.3 tests were dropped too.

Reply:
It has, indeed.